.github/dependabot.yml

```diff
@@ -0,0 +1,7 @@
+version: 2
+updates:
+  - package-ecosystem: 'bundler'
+    directory: '/'
+    schedule:
+      interval: 'daily'
+    open-pull-requests-limit: 10
```

Comment:
is that necessary? we don't set it in the other repos and so far didn't have any PR floods.

Reply:
https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file#open-pull-requests-limit says the default limit is 5. I don't think we need to increase that to 10 so :+1: for removing this part.